runtime/compiler/env/j9methodServer.cpp

```diff
@@ -152,6 +152,15 @@ TR_ResolvedJ9JITServerMethod::staticAttributes(TR::Compilation * comp, I_32 cpIn
    return result;
    }
 
+TR_OpaqueClassBlock * 
+TR_ResolvedJ9JITServerMethod::definingClassFromCPFieldRef(TR::Compilation *comp, int32_t cpIndex, bool isStatic)
+   {
+   // Send a message to the client
+   _stream->write(JITServer::MessageType::ResolvedMethod_definingClassFromCPFieldRef, _remoteMirror, cpIndex, isStatic);
```

Comment:
I don't understand.

Reply:
Sorry, my mistake, I thought the code in `TR_ResolvedJ9Method::definingClassFromCPFieldRef()` that we were trying to avoid executing on the server was inside an `if (isStatic)` check and that for `isStatic == false` we could still call the original implementation rather than going over the network, but it's not.